README.rst

```diff
@@ -6,8 +6,9 @@ Python Enhancement Proposals
 
 The PEPs in this repo are published automatically on the web at
 https://www.python.org/dev/peps/.  To learn more about the purpose of
-PEPs and how to go about writing a PEP, please start reading at PEP 1
-(``pep-0001.txt`` in this repo).  Note that PEP 0, the index PEP, is
+PEPs and how to go about writing one, please start reading at `PEP 1
+<https://www.python.org/dev/peps/pep-0001/>`_.
+Note that PEP 0, the index PEP, is
 now automatically generated, and not committed to the repo.
```

Comment:
It has been generated for many years, so remove "now"
```suggestion
automatically generated, and not committed to the repo.
```

Reply:
Timeless documentation, yep. Done.